tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>Taking a subset of data lacks context. In QAT, it's mentioned to demonstrate finetuning. In pruning, I've removed it since finetuning is not effectively enough to just use 1000 images. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>yep, I agree.</p><p>Also, the epochs = 1 seems problematic. The loss is still high. Did we want it to be 1 so that it runs faster?</p>

Reply:
Epochs = 1 isn't not essential. I'd pick something that is just a reasonable balance between minimizing loss and training time. If the # of epochs is more than it took to train than the baseline model, then that's where it gets weird because "finetuning" may no longer be the right way to put it.

Reply:
<p>sorry, this is a wrong comment from me. </p><p>I wanted to comment on the epochs in the baseline model training of QAT, which was set to be 1. Then, I saw you set it to be 4 in the pruning example so it is fine.</p><p>Let's ignore this.</p>